Do not build the jiffies by default, they are not intended for
distribution.
--- tvips/CMakeLists.txt.orig
+++ tvips/CMakeLists.txt
@@ -171,6 +171,9 @@ add_definitions(-D_DARWIN_C_SOURCE)
 # The executables should generally depend on the _update_config_h
 # target.  XXX Oddly have to give libm for dan_test, imgcorr, and
 # rawdump, but nothing else?
+set(BUILD_JIFFIES OFF CACHE BOOL
+  "Build MicroED developer jiffies, not for distribution")
+if(BUILD_JIFFIES)
 add_executable(cphdr
   "cphdr.c"
   "frame.c")
@@ -181,6 +184,7 @@ add_executable(dan_test
   "frame.c")
 add_dependencies(dan_test _update_config_h)
 target_link_libraries(dan_test m)
+endif()
 
 add_executable(dumpframe
   "compat.c"
@@ -193,6 +197,7 @@ add_executable(dumpframe
 add_dependencies(dumpframe _update_config_h)
 target_link_libraries(dumpframe ${TIFF_LIBRARIES})
 
+if(BUILD_JIFFIES)
 add_executable(fixpoint
   "delta_t.cpp"
   "fixpoint.c")
@@ -246,6 +251,7 @@ add_executable(tiff2raw
   "util.c")
 add_dependencies(tiff2raw _update_config_h)
 target_link_libraries(tiff2raw ${TIFF_LIBRARIES})
+endif()
 
 add_executable(tiff2smv
   "compat.c"
@@ -350,19 +356,22 @@ add_test(tvips2smv ${CMAKE_COMMAND}
 # CMake is generating the build system.  If no documentation exists,
 # do nothing.
 install(
-  TARGETS tiff2smv tvips2smv
+  TARGETS
+    dumpframe
+    tiff2smv
+    tvips2smv
   DESTINATION bin)
 install(
   FILES
     "${PROJECT_SOURCE_DIR}/tiff2smv.1"
     "${PROJECT_SOURCE_DIR}/tvips2smv.1"
-  DESTINATION "${CMAKE_INSTALL_FULL_MANDIR}/man1"
+  DESTINATION "${CMAKE_INSTALL_MANDIR}/man1"
   OPTIONAL)
 install(
   FILES
     "${PROJECT_BINARY_DIR}/tiff2smv.1"
     "${PROJECT_BINARY_DIR}/tvips2smv.1"
-  DESTINATION "${CMAKE_INSTALL_FULL_MANDIR}/man1"
+  DESTINATION "${CMAKE_INSTALL_MANDIR}/man1"
   OPTIONAL)
 
 
